ops/metrics.go

```diff
@@ -0,0 +1,156 @@
+package ops
+
+import (
+	"context"
+	"strings"
+
+	"github.com/rs/zerolog/log"
+	"go.opentelemetry.io/contrib/instrumentation/runtime"
+	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp"
+	"go.opentelemetry.io/otel/exporters/stdout/stdoutmetric"
+	"go.opentelemetry.io/otel/metric/global"
+	"go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/resource"
+	"go.opentelemetry.io/otel/semconv/v1.17.0"
+
+	"github.com/valkyrie-fnd/valkyrie/configs"
+)
+
+type MetricExporterType string
+
+const (
+	MetricStdOut   MetricExporterType = "stdout"
+	MetricOTLPHTTP MetricExporterType = "otlpmetrichttp"
+	MetricNone     MetricExporterType = ""
+)
+
+// noMetricConfig default empty MetricConfig
+var noMetricConfig = MetricConfig{}
+
+type MetricConfig struct {
+	Exporter MetricExporterType
+	Version  string
+	configs.MetricConfig
+	configs.TelemetryConfig
+}
+
+// ConfigureMetrics configures metrics, including exporter and instrumentation
+func ConfigureMetrics(vConf *configs.ValkyrieConfig) error {
+	cfg := metricConfig(vConf)
+
+	// No config - no setup
+	if *cfg == noMetricConfig {
+		return nil
+	}
+
+	exp, err := createMetricExporter(cfg)
+	if err != nil {
+		log.Warn().Err(err).Msg("Failed to setup metric exporter")
+		return err
+	}
+
+	// labels/tags/resources that are common to all metrics.
+	resource := resource.NewWithAttributes(
+		semconv.SchemaURL,
+		semconv.ServiceName(cfg.ServiceName),
+		semconv.ServiceNamespace(cfg.Namespace),
+		semconv.ServiceVersion(cfg.Version),
+	)
+
+	mp := metric.NewMeterProvider(
+		metric.WithResource(resource),
+		metric.WithReader(
+			// collects and exports metric data every 60 seconds by default.
+			metric.NewPeriodicReader(exp),
+		),
+	)
+
+	global.SetMeterProvider(mp)
+
+	log.Info().Msg("Configured metrics")
```

Comment:
This log entry should 
- contain more info, like what exporter type was configured
- or be debug level
- or be removed 🧹